simtools/simtel/simtel_io_histogram.py

```diff
@@ -207,19 +218,25 @@ def fill_event_histogram_dicts(self):
         self._logger.error(msg)
         raise HistogramIdNotFoundError
 
-    @property
-    def view_cone(self):
+    def _set_view_cone(self):
         """
         View cone used in the simulation.
 
-        Returns
+        Raises
         -------
-        list:
-            view cone used in the simulation [min, max]
-        """
-        if self._view_cone is None:
-            self._view_cone = self.config["viewcone"] * u.deg
-        return self._view_cone
+        ValueError:
+            if input parameter is missing.
+        """
+        if self.view_cone is None:
+            try:
+                self.view_cone = self.config["viewcone"] * u.deg
+            except TypeError as exc:
+                msg = "view_cone needs to be passed as argument (a list of cone in deg)."
```

Comment:
`a list of cone` is unclear. "Minimum and maximum of the viewcone radius in deg"

Reply:
Done.